src/browser/web-view.js

```diff
@@ -260,7 +260,11 @@ define((require, exports, module) => {
       onTitleChange: event => edit(WebView.setTitle(event.detail)),
       onPrompt: event => console.log(event),
       onAuthentificate: event => console.log(event),
-      onScrollAreaChange: state.contentOverflows == void(0) && (event =>
+      // This will trigger a resize. If the content react to the resize by changing its
+      // layout, this might change the scrollarea again, triggering a resize… infinite
+      // loop.
+      // So we only allow contentOverflows to transition from false (default value) to true.
+      onScrollAreaChange: !state.contentOverflows && (event =>
```

Comment:
I tried and can't reproduce.

If there's anything strange happening, the only thing I could see is a weird behavior when we reset `contentOverflows` on load.